packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -418,10 +410,9 @@ Directory (UD) isn't used to store the tenants’ users and groups. See below
 for an illustrated example of this
 configuration.
 
-<div class="image-at-80">
+<div class="three-quarter">
```

Comment:
I think this could go to half

Reply:
I disagree. I think too much of the detail is hard to read at half size. Keeping as-is.